target/linux/ramips/dts/mt7621_mikrotik_routerboard-7xx.dtsi

```diff
@@ -40,29 +40,44 @@
 
 &switch0 {
 	ports {
-		port@0 {
+		port0: port@0 {
 			status = "okay";
 			label = "wan";
+
+			nvmem-cells = <&macaddr_hard 0>;
+			nvmem-cell-names = "mac-address";
 		};
 
 		port@1 {
 			status = "okay";
 			label = "lan2";
+
+			nvmem-cells = <&macaddr_hard 1>;
+			nvmem-cell-names = "mac-address";
 		};
 
 		port@2 {
 			status = "okay";
 			label = "lan3";
+
+			nvmem-cells = <&macaddr_hard 1>;
+			nvmem-cell-names = "mac-address";
 		};
 
 		port@3 {
 			status = "okay";
 			label = "lan4";
+
+			nvmem-cells = <&macaddr_hard 1>;
+			nvmem-cell-names = "mac-address";
 		};
 
 		port@4 {
 			status = "okay";
 			label = "lan5";
+
+			nvmem-cells = <&macaddr_hard 1>;
+			nvmem-cell-names = "mac-address";
 		};
```

Comment:
If we are setting these MACs, could we please increment them for each port (2, 3, 4), as done in OEM software. That way if the ports are used removed from the default bridge they have their own MAC address. I believe bridge uses MAC of first port added, so that will not change.

These Mikrotik device labels typically show the first and last MAC address, so can confirm allocation via device photos that show label. [760igs](https://a.allegroimg.com/original/1e2614/f2d564a7424ebf3296c91aeab2fc) [750gr3](https://cf.shopee.co.id/file/id-11134201-23020-11qnpltxhnnv70)